.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.9'
+          python-version: '3.11'
```

Comment:
I wonder if we could update this to 3.10 instead of 3.11? 3.10 is what GRR is uses and it would be great to have a uniform Python version in GRR and Fleetspeak.

Reply:
Sure I don't see why not. The reason I chose 3.11 is because that's what's installed on my development VM where 3.9 is not available, but 3.10 is afaik.